### DISCUSSION TITLE
Tuya Alarm Siren - any one got it to work ?

### DISCUSSION THREAD

**Discussion by jonandel** [Koenkk/zigbee2mqtt#7686] [Devices]
I bought this 
https://www.aliexpress.com/item/1005002024275074.html?spm=a2g0s.9042311.0.0.54bf4c4dDbftD1

Thinking its the same as described in the ZB2MQTT Supported device list https://www.zigbee2mqtt.io/devices/TS0216.html

But it has arrived and has no volume buttons, and it ZB2MQTT (standard and Edge) dont allow it to pair.
Anyone got it working ?

**Comment by iglu-sebastian** [comment]
@jonandel I rewrote some support for TS0216, works fine but mine has buttons, maybe new model? Did it come with any instructions?

**Comment by jonandel** [comment]
My apologies, I posted the wrong item.  My TS0216 is working fine.

 

I meant to post/ask about this one (not on compatibilities list), but ZB 3.0 :

https://www.aliexpress.com/item/1005002718848381.html
